pkg/insights/insightsclient/insightsclient.go

```diff
@@ -229,6 +225,17 @@ func (c *Client) Send(ctx context.Context, endpoint string, source Source) error
 	return nil
 }
 
+func responseBody(r *http.Response) string {
+	if r == nil {
+		return ""
+	}
+	body, _ := ioutil.ReadAll(r.Body)
```

Comment:
Is it OK to ignore the error here?

Reply:
I think so, see the original version. We are trying to log what we received from response. If we cannot, we will log empty string.

Reply:
Ahh yes...sorry I was looking rather at the green fields than the red ones.